lang/bash/.transpile

```diff
@@ -139,7 +145,7 @@ transpile_array_push(){
 }
 
 transpile_array_get(){
-  code="${1/\$/\"\$\{}"
+  code="${1/\$/\"\${}"
```

Comment:
Does this cause problems as well? The bash in travis wasn't handling the escapes correctly so I removed as many as I could, but if it's causing problems I will look for another way of fixing it.